src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -84,6 +106,9 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, location
         if resourceTypes.resource_type == 'connectedClusters':
             rp_locations = [location.replace(" ", "").lower() for location in resourceTypes.locations]
             if location.lower() not in rp_locations:
+                telemetry.set_user_fault()
+                telemetry.set_exception(exception='Location not supported', fault_type='location-validation-error',
+                                        summary='Provided location is not supported for creating connected clusters')
                 raise CLIError("Connected cluster resource creation is supported only in the following locations: " +
```

Comment:
>default [](start = 25, length = 7)

Actually, the param can be auto-detect. For telemetry we should put what we detected. If we dont detect (like today) we should trace/ log default. Hence I think default as distro is better.